src/mgmt/deploy-director-vm.sh

```diff
@@ -336,8 +322,8 @@ rm -f /store/data/images/director.img
     --vcpus 8 \
     --hvm \
     --os-type linux \
-    --os-variant rhel7 \
-    --disk /store/data/images/director.img,bus=virtio,size=80 \
+    --os-variant rhel8.1 \
```

Comment:
Do we have to spell out the sub version, e.g, rhel8.1 vs rhel8?  Seems like we should stick with just rhel8.

Reply:
we don't have to be specific. will just change it to 8 then